<?xml version="1.0"?>
<launch>
    <node pkg="velodyne_pointcloud" type="cloud_node" name="convert_veloscans">
	<param name="calibration" value="$(find ndt_fuser)/launch/velo32.yaml" />
    </node>
    <node pkg="ndt_fuser" type="ndt_fuser_node" name="fuser_3d" output="screen">
	<param name="points_topic" value="/velodyne_points" />
	<param name="matchLaser" value="false" />
	<param name="size_x_meters" value="200" />
	<param name="size_y_meters" value="200" />
	<param name="size_z_meters" value="6" />
	<param name="resolution" value="0.5" />
	<param name="sensor_range" value="40." />
	
	<!-- Specific sensor offset parameters for the example file -->
	<param name="sensor_pose_x" value="1.18" />
	<param name="sensor_pose_y" value="-0.3" />
	<param name="sensor_pose_z" value="2.0" />
	<param name="sensor_pose_t" value="-1.6" />
	
	<param name="map_directory" value="$(find ndt_fuser)/maps/" />	
	<param name="map_name_prefix" value="arla" />
	
	<param name="beHMT" value="false" />
	<param name="match2D" value="true" />
	<param name="useOdometry" value="true" />
	<param name="odometry_topic" value="/vmc_navserver/odom" />
	<param name="plotGTTrack" value="true" />
	<param name="initPoseFromGT" value="true" />
	<param name="gt_topic" value="/vmc_navserver/state" />
	<param name="visualize" value="true" />
    </node>
</launch>
